source/extensions/filters/common/expr/library/custom_functions.cc

```diff
@@ -0,0 +1,67 @@
+#include "source/extensions/filters/common/expr/library/custom_functions.h"
+
+#include "source/common/protobuf/protobuf.h"
+
+#include "eval/public/cel_value.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Filters {
+namespace Common {
+namespace Expr {
+namespace Library {
+
+absl::Status GetProductCelFunction::Evaluate(absl::Span<const CelValue> args, CelValue* output,
+                                             Protobuf::Arena* arena) const {
+  // using arena so that it will not be unused
+  arena->SpaceUsed();
+  if (args[0].type() == CelValue::Type::kInt64 && args[1].type() == CelValue::Type::kInt64) {
```

Comment:
Here and below check args size as well?

Reply:
It seems this checking (number of args and their types) is done for us.